react/react1/week3/todo-app/src/components/Container/Container.jsx

```diff
@@ -0,0 +1,5 @@
+const Container = ({ children }) => {
+  return <div>{children}</div>;
```

Comment:
Good use of children, but maybe add some styling to the div to better visualise how we could use wrapper components in practice